extensions/amp-story/1.0/amp-story.js

```diff
@@ -1090,28 +1131,37 @@ export class AmpStory extends AMP.BaseElement {
   }
 
   /**
-   * Displays the unsupported browser UI: either the publisher provided UI, or
-   * fallbacks to a generic default.
+   * Displays changes to the support state of the browser
    * @param {boolean} isBrowserSupported
    * @private
    */
   onSupportedBrowserStateUpdate_(isBrowserSupported) {
-    if (isBrowserSupported) {
-      dev().error(TAG, 'No handler to exit unsupported browser state.');
-    }
-
     const fallbackEl = this.getFallback();
-
-    this.mutateElement(() => {
-      this.element.classList.add('i-amphtml-story-fallback');
-    });
-
-    // Displays the publisher provided fallback, or fallbacks to the default
-    // unsupported browser layer.
-    if (fallbackEl) {
-      this.toggleFallback(true);
+    if (isBrowserSupported) {
+      // Removes the default unsupported browser layer or throws an error
+      // if the publisher has provided their own fallback
+      if (fallbackEl) {
+        dev().error(TAG, 'No handler to exit unsupported browser state on ' +
+        'publisher provided fallback.');
```

Comment:
The fallback just replaces the default "unsupported browser" UI, right? So we should hide the fallback and display the story in this case.

Try testing this manually to make sure it works (w/ and w/o fallback).

Reply:
I was thinking that since the "Continue Anyway" button is only on the default unsupported UI that you mentioned, then if a publisher provided their own page, they would not be able to change the browser support state to supported.

Before my changes, there was a dev().error stating that there can't be a way to change a browser support state from false to true, so I thought using the button on the default page was the only way it could happen, and I would return a similar error otherwise.

Unless can they specify their own "Continue Anyway" button or something similar to that to toggle? Sorry for my confusion!

Reply:
Good point. 😄 @gmajoulet should we also display the "Continue anyways" button on publisher fallback?

Reply:
I actually did have a conversation about this with him earlier as we were thinking of implementing support for the publisher provided fallbacks. At that time he told me for now at least we should just support it for the provided fallback. 

He told me to add a comment specifying this, so I did so in lines 50-52 of the amp-story-unsupported-browser-layer.js file, so please let me know if I should add it elsewhere instead/as well!

Reply:
+1 to only adding this to our fallback for now; seems much less of a problem for publisher-provided fallback.  If they so choose, publishers can provide a fallback version of the actual content, instead of just a deadend "we don't support you" screen.